_features/discoverable.html

```diff
@@ -2,34 +2,22 @@
 title: Discoverable
 slug: discoverable
 description: Are identifiable as applications and search engines can find them.
+usecases:
+- <p>Progressive Web Applications offer unique capabilities; as a developer, I want
+  my web applications to be promoted in search engines when the user is using
+  modern platforms.</p>
```

Comment:
Hmm, "modern platforms" isn't clear. I assume "platform" here means operating system platform—Windows, Mac, Linux—but it's actually the browser that needs to be "modern" enough for progressive apps to benefit from the progressive APIs.

Although I'm not sure even that is necessary in order for progressive apps to be promoted in search engines. That could happen even in legacy browsers, since it's a function of the search engine.

This minimal change here is to replace "modern platforms" with "modern browsers." But I'd go further and strip "when the user is using modern platforms," unless we think search engines won't highlight apps on older browsers.